eng/Versions.props

```diff
@@ -161,6 +161,10 @@
     <MicrosoftTestPlatformCLIPackageVersion>17.13.0-preview-25055-01</MicrosoftTestPlatformCLIPackageVersion>
     <MicrosoftTestPlatformBuildPackageVersion>17.13.0-preview-25055-01</MicrosoftTestPlatformBuildPackageVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <!-- Dependencies from https://github.com/Microsoft/testfx -->
+    <MicrosoftTestingPlatformPackageVersion>1.7.0-dev</MicrosoftTestingPlatformPackageVersion>
```

Comment:
I think this should not be commited as-is (it contains a `-dev`). Besides we are currently working on `1.6.0-preview` at the moment.